oci-objectstorage/src/main/scala/tamer/oci/objectstorage/OciObjectStorageConfiguration.scala

```diff
@@ -29,6 +29,7 @@ final case class OciObjectStorageConfiguration[
     namespace: String,
     bucketName: String,
     prefix: Option[String],
+    objectNameFinder: String => Boolean,
```

Comment:
nit
```suggestion
    objectNameFilter: String => Boolean,
```

Reply:
It's not strictly a filter though, it's a finder that stops at the first match.

Reply:
yes, you're absolutely right